02_ShowUser.php

```diff
@@ -2,112 +2,110 @@
 <html lang = "eng">
 	<head>
 		<title>Sound Wave</title>
-        <meta charset = "utf-8">
+		<meta charset = "utf-8">
+
+		<!-- styles -->
 		<link rel = "stylesheet" type = "text/css" href = "css/fonts.css">
 		<link rel = "stylesheet" type = "text/css" href = "css/grids.css">
-		<link rel = "stylesheet" href = "css/style.css"/>
-		<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
 		<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
 		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
 		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 		<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
 
-		
-        <meta name = "Keywords" content = "Graeme, Music Database, Soundwave, Music, Tunes"/>
-        <meta name = "Author" content = "Liv Amer"/>
-        <meta name = "Description" content = "Graeme's Music Database"/>
-        <meta name = "viewport" content = "width=device-width,initial-scale=1"/>
-        
+		<meta name = "Keywords" content = "Graeme, Music Database, Soundwave, Music, Tunes"/>
+		<meta name = "Author" content = "Liv Amer"/>
+		<meta name = "Description" content = "Graeme's Music Database"/>
+		<meta name = "viewport" content = "width=device-width,initial-scale=1"/>
+
 
 	</head>
-		<body>
-		
-			<!-- Navbar (sit on top) -->
-			<div class="w3-top">
-			 	<div class="w3-bar w3-white w3-wide w3-padding w3-card">
-					<img src="images/placeholder.jpg" width="40" height="40">
-					<a href="index.html" class="w3-bar-item w3-button"><b>logo</b> </a>
-	  
-		 <!-- Float links to the right -->
-					<div class="w3-right" >
-					  <a href="indexV1.php" class="w3-bar-item w3-button w3-xxlarge fa fa-home"></a>
-					  <a href="playlistV1.php" class="w3-bar-item w3-button w3-xxlarge material-icons">playlist_add</a>	
-					  <a href="contact.php" class="w3-bar-item w3-button w3-xxlarge material-icons" >contact_support</a>
-					  <a href="settings.php" class="w3-bar-item w3-button w3-xxlarge material-icons">settings</a>	
-    				</div>
+	<body>
+
+		<!-- Navbar (sit on top) -->
+		<div class="w3-top">
+			<div class="w3-bar w3-grey w3-padding w3-card">
```

Comment:
changed this class by removing the white in exchange for the grey to fit the theme, an also removing the effect of having the font wide within the navbar; preferred to have it as it normal.